sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -1159,6 +1167,62 @@ public void testWriting() throws Exception {
     assertEquals(ByteString.copyFromUtf8(value), rows.get(ByteString.copyFromUtf8(key)));
   }
 
+  /** Tests that at least one result is emitted per element written in the global window. */
```

Comment:
Please add a test that demonstrates usage of this API with Wait.on(), as this is its primary use case.